tools/generate_hotkeys.py

```diff
@@ -6,20 +6,14 @@
 from zulipterminal.config.keys import HELP_CATEGORIES, KEY_BINDINGS
```

Comment:
Re this commit text, you shouldn't be to refer to the previous git history because once it's merged it's like it never existed so it wouldn't make sense unless it's for the reviewer. You should mostly talk about what this commit is doing.